src/components/Info/_info.scss

```diff
@@ -0,0 +1,51 @@
+.info-section__heading {
+  @include carbon--type-style('heading-01');
+}
+.info-card {
+  margin-top: $spacing-09;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
This closing bracket should be after the `@include carbon--breakpoint(lg)` class

Reply:
updated per request

Reply:
This closing bracket needs to be removed to resolve the grey left border that is showing up for all your divs (see your preview: https://deploy-preview-2079--carbon-tutorial.netlify.com/#/)